tools/pyi/gen_pyi.py

```diff
@@ -123,6 +123,7 @@ def type_to_python(typename, size=None):
         'int64_t': '_int',
         'accreal': 'Number',
         'real': 'Number',
+        'QScheme': 'QScheme',
```

Comment:
this doesn't seem consistent, "QScheme" isn't a python type; I don't know what's going on with dtype here though, you might want to track that down.

Reply:
removed